vendor/engines/production_credits/app/controllers/production_credits/venues_controller.rb

```diff
@@ -0,0 +1,12 @@
+module ProductionCredits
+  class VenuesController < ApplicationController
+    skip_before_action :verify_authenticity_token
+    skip_before_filter :authenticate_user!
+
+    def index
+      venues = Venue.for_production_ids(params[:production_ids])
```

Comment:
you could skip this step if you already know there are no production_ids

```
venues = Venue.for_production_ids(params[:production_ids]) if params[:production_ids]
```
Or some variation of that conditional.